scene/2d/particles_2d.h

```diff
@@ -48,7 +48,10 @@ class Particles2D : public Node2D {
 private:
 	RID particles;
 
-	bool one_shot;
+	bool emitting = false;
+	bool active = false;
+	bool signal_cancled = false;
```

Comment:
```suggestion
	bool signal_canceled = false;
```
Same thing for 3D.

There was the same typo in the original PR, I just fixed it in 5c43e4c1efc85856a4e918bc67e1266657b9110c.